x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/service_page/form_row_select.tsx

```diff
@@ -5,23 +5,23 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
 import {
   EuiDescribedFormGroup,
-  EuiSelectOption,
+  EuiComboBoxOptionOption,
   EuiFormRow,
+  EuiComboBox,
 } from '@elastic/eui';
-import { SelectWithPlaceholder } from '../../../../../shared/select_with_placeholder';
-
 interface Props {
   title: string;
   description: string;
   fieldLabel: string;
   isLoading: boolean;
-  options?: EuiSelectOption[];
+  options: Array<EuiComboBoxOptionOption<string>>;
```

Comment:
@kpatticha is now options always defined?

Reply:
fixed it here https://github.com/elastic/kibana/pull/122648/commits/980356bd6ea81ab9dd46d3d113c03058a6607b57